docs/StudentHealthTracking.md

```diff
@@ -21,7 +21,7 @@ A custom Vitals dropdown field (`vital_c`) was added to the Students module and
 
 When `deceased` is selected, a new Death Information panel appears on the Record view and users are required to enter a Cause of Death (`cause_of_death_c`).
 
-Professor M can then build reports based upon vital status and the common causes of death.
+Two reports in a dev instance and then exported via REST API. These reports are then added to the Prof M. data collection. These reports rely primarily on `vital_c` and `cause_of_death_c` fields. 
```

Comment:
This sentence doesn't make sense to me: Two reports in a dev instance and then exported via REST API.